superset/models/core.py

```diff
@@ -993,7 +993,7 @@ def get_schema_access_for_file_upload(  # pylint: disable=invalid-name
                 self, g.user
             )
             allowed_databases += extra_allowed_databases
-        return sorted(set(allowed_databases))
```

Comment:
@betodealmeida Do you know why it was sorted before?

Reply:
It was sorted so it can be displayed nicely in the UI, but since that's a UI-only concern we've been moving the sorting logic to the frontend. I'm going to update it next, for now this PR fixes only this because it's broken by returning a list.

Reply:
Should the frontend change also happen in this PR to make it atomic? This would make the PR not only fix the list issue but also preserve the current sorted behavior.